terraso_backend/tests/graphql/conftest.py

```diff
@@ -186,9 +186,7 @@ def data_entries(users, groups):
     creator = users[0]
     creator_group = groups[0]
     creator_group.members.add(creator)
-    return mixer.cycle(5).blend(
-        DataEntry, slug=None, created_by=creator, size=100, groups=creator_group
-    )
+    return mixer.cycle(5).blend(DataEntry, created_by=creator, size=100, groups=creator_group)
```

Comment:
I have a question here for my own understanding: why can you remove the `slug=None` here? Looking at the source, I don't see any changes to the `DataEntry` object.